subsys/bluetooth/host/audio/chan.c

```diff
@@ -1023,14 +1023,20 @@ int bt_audio_cig_terminate(struct bt_audio_chan *chan)
 		return -EINVAL;
 	}
 
-	for (int i = 0; i < ARRAY_SIZE(cigs); i++) {
+	for (int i = 0; i < ARRAY_SIZE(unicast_groups); i++) {
+		int err;
 		struct bt_iso_cig *cig;
 
-		cig = cigs[i];
+		cig = unicast_groups[i].cig;
 		if (cig != NULL &&
 		    cig->cis != NULL &&
 		    cig->cis[0] == chan->iso) {
-			return bt_iso_cig_terminate(cig);
+			err = bt_iso_cig_terminate(cig);
+			if (err == 0) {
```

Comment:
(Seems like an unrelated bugfix.  Next time, I would prefer that as a separate commit.  OK for now.)

Reply:
It is somewhat unrelated, so could have been a different commit.